packages/components/src/button/types.ts

```diff
@@ -45,10 +45,6 @@ type BaseButtonProps = {
 	/**
 	 * Whether the button is focused.
 	 */
-	focus?: boolean;
-	/**
-	 * If provided, renders an Icon component inside the button.
-	 */
```

Comment:
This description text refers to the `icon` prop, and should not be deleted. We should detele, instead, the lines above the prop ("Whether the button is focused." text)